src/test/java/com/dummyblog/app/service/impl/PostServiceImplTest.java

```diff
@@ -27,47 +26,59 @@ public void init() {
 	    modelMapper = Mockito.mock(ModelMapper.class);
 		underTest = new PostServiceImpl(postRepository, modelMapper);
 	}
-
+	
 	@Test
 	public void testGetPostsShouldCallPostRepository() {
 		// Given
-		List<PostDto> expected = List.of(new Post(), new Post()).stream().map(post -> modelMapper.map(post, PostDto.class)).collect(Collectors.toList());
-		Mockito.when(postRepository.findAll().stream().map(post -> modelMapper.map(post, PostDto.class)).collect(Collectors.toList())).thenReturn(expected);
+		PostDto postDto1 = Mockito.mock(PostDto.class);
+		PostDto postDto2 = Mockito.mock(PostDto.class);
+		List<PostDto> expected = List.of(postDto1, postDto2);
+		Post post1 = Mockito.mock(Post.class);
+		Post post2 = Mockito.mock(Post.class);
+		Mockito.when(postRepository.findAll()).thenReturn(List.of(post1, post2));
+		Mockito.when(modelMapper.map(post1, PostDto.class)).thenReturn(postDto1);
+		Mockito.when(modelMapper.map(post2, PostDto.class)).thenReturn(postDto2);
 		
 		// When
 		List<PostDto> actual = underTest.getPosts();
 		
 		// Then
 		Assertions.assertEquals(expected, actual);
 		Mockito.verify(postRepository).findAll();
-		Mockito.verifyNoMoreInteractions(postRepository);
+		Mockito.verify(modelMapper).map(post1, PostDto.class);
+		Mockito.verify(modelMapper).map(post2, PostDto.class);
+		Mockito.verifyNoMoreInteractions(postRepository, modelMapper, postDto1, postDto2, post1, post2);
 	}
 
 	@Test
-	public void testGetPostShouldThrowNullPointerExceptionWhenTheTitleParameterIsNull() {
+	public void testGetPostShouldReturnAPostDtoWhenThePostExists() {
 		// Given
+		PostDto expected = Mockito.mock(PostDto.class);
+		Post post = Mockito.mock(Post.class);
+		Mockito.when(postRepository.findByTitle(THE_TITLE)).thenReturn(post);
+		Mockito.when(modelMapper.map(post, PostDto.class)).thenReturn(expected);
 		
 		// When
-		Assertions.assertThrows(NullPointerException.class, () -> underTest.getPost(null));
+		PostDto actual = underTest.getPost(THE_TITLE);
 		
 		// Then
-		Mockito.verifyNoMoreInteractions(postRepository);
+		Assertions.assertEquals(expected, actual);
+		Mockito.verify(postRepository).findByTitle(THE_TITLE);
+	    Mockito.verify(modelMapper).map(post, PostDto.class);
```

Comment:
Ebben a sorban megint vegyesen van tab és szóköz használva indentálásra. A sor elején egy tab és utána 2 szóköz. Emiatt különböző szerkesztőkben szétcsúszhat a kód. IDE-ből lehet hogy ez nem látszik, de pl már itt GitHub-on is kicsit szétcsúszik. Crtl+Shift F-fel tudod formázni a kódot Eclipse-ben, azzal ezt is meg tudod könnyen formázni.
Eclipse default-ből tab-ot használ, viszont úgy általánosságban a 4 szóköz elterjedtebb. Ahogy korábban írtam különösebben egyik sem jobb a másiknál (sok dev-nél vita tárgya, mert vannak pro-k és konktrák), a lényeg, hogy döntsd el neked melyik a szimpatikusabb és következetes azt használd.

Reply:
Így már értem. Csodálkoztam is, hogy én nem láttam azokból a hibákból párat, amit jeleztél. Most végig mentem mindenen Ctrl+Shift+F-fel.